frontend/scss/ionic.app.scss

```diff
@@ -28,3 +28,7 @@ $ionicons-font-path: "../lib/ionic/fonts" !default;
 @import "www/lib/ionic/scss/ionic";
 @import "style";
 @import "cooperatives";
+.disable-clickon-tabs {
```

Comment:
ionic.app.scss is normall just used to import other files. Better place for this code would be in style.scss